src/compiler/types.ts

```diff
@@ -6909,10 +6924,8 @@ export interface Diagnostic extends DiagnosticRelatedInformation {
     /** @internal */ skippedOn?: keyof CompilerOptions;
 }
 
-/** @internal */
 export type DiagnosticArguments = (string | number)[];

```

Comment:
This stuff definitely should not become public.

Reply:
OK, let’s keep `any` for the `ErrorCallback` then